tests/rustdoc_edge_cases.rs

```diff
@@ -233,9 +233,4 @@ fn release_type_flag_minor() {
         ])
         .assert()
         .failure();
-    let mut cmd = Command::cargo_bin("cargo-semver-checks").unwrap();
-    cmd.current_dir("test_crates/type_marked_deprecated/new")
-        .args(["semver-checks", "check-release", "--baseline-root=../old"])
-        .assert()
-        .failure();
```

Comment:
Why are we deleting this test?

At minimum, the comment on line 225 needs updating. If possible & reasonable, I'd like to preserve this test and just update it to test the new behavior.

Reply:
Yeah, I agree., I should not delete this.Thee reason I delete it before is that its behavior it is the same as the above test, but yeah we should keep it for clarity
```rs
    let mut cmd = Command::cargo_bin("cargo-semver-checks").unwrap();
    cmd.current_dir("test_crates/type_marked_deprecated/new")
        .args([
            "semver-checks",
            "check-release",
            "--baseline-root=../old",
            "--release-type=minor",
        ])
        .assert()
        .success();
```